addons/api/mirage/models/scope.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model from './base';
+
+export default Model.extend({});
```

Comment:
without this scope model file some values (populated by factories) were not getting serialized properly.(ex. missing created_time values).